frontend/www/js/omegaup/arena/coursev2.ts

```diff
@@ -23,8 +23,8 @@ OmegaUp.on('ready', async () => {
     },
     data: () => {
       return {
-        selectedTab: null as null | string,
-        problemSelectedTab: null as null | string,
+        selectedTab: null as Tabs | string,
+        problemSelectedTab: null as ProblemTabs | string,
```

Comment:
esto era al revés, no?

```suggestion
        selectedTab: null as null | Tabs,
        problemSelectedTab: null as null | ProblemTabs,
```

Reply:
Ya está, pero sigue fallando. Quizás debo poner `ProblemTabs.Entry1 | ProblemTabs.Entry2 | ... | ProblemTabs.EntryN` ??

Reply:
Estoy regresando el tipo a `string` porque no he encontrado otra solución :/